src/extension/features/accounts/assisted-clear/components/AssistedClearContainer.jsx

```diff
@@ -29,6 +29,11 @@ export const ClearAssistantContainer = ({ reconcileInputValue }) => {
     let unclearedTransactions = transactions.filter(
       txn => txn.cleared && txn.isUncleared() && !txn.isTombstone
     );
+
+    // Note: For credit cards, we'll automatically invert to follow ynab's behavior
+    if (reconcileInputValue > 0 && account.getAccountType() == 'CreditCard') {
+      reconcileInputValue *= -1;
+    }
     let calculatedTarget = Number(reconcileInputValue) * 1000 - clearedBalance;
```

Comment:
You should probably use the `export function formatCurrency(valueInMilliDollars, hideSymbol)` to format the number. That function calls a YNAB function to do the formating so if they change things, we're protected. It's located in src/extension/utils/currency.js

Reply:
I ended up using `stripCurrency` currency instead. It parses the string (input value) and converts it to millidollars as a number type. This also ends up solving https://github.com/toolkit-for-ynab/toolkit-for-ynab/issues/2235 as well. Thanks! 👍 👍